### ISSUE TITLE
0.5.0 yarn install results Uncaught TypeError: (0 , _marked.marked) is not a function

### ISSUE THREAD

**Issue by gastongouron** [wingkwong/react-quiz-component#82]
Hello, thank you for your work and the lib whose features looks very nice!

After running through previous issues concerning this error ( [79](https://github.com/wingkwong/react-quiz-component/issues/79) and [35](https://github.com/wingkwong/react-quiz-component/issues/35) ), I'm facing this issue even tho I use 0.5.0 after a yarn install.

Fresh install, nothing fancy:
```
yarn add react-quiz-component@0.5.0
```

Simple component as per the readme as :
```
...
const quiz = {...}
...
return <Quiz quiz={quiz}/>
...
```

results in:
<img width="686" alt="Screenshot 2022-12-08 at 15 52 50" src="https://user-images.githubusercontent.com/1798822/206480508-7cf160fc-6a60-4b39-b2db-f6834b867c3c.png">

I also tried:
```
yarn add react-quiz-component@https://github.com/wingkwong/react-quiz-component.git\#develop
```
(which seems to fail for other reasons)

Any input truly appreaciated.

**Comment by SaadBazaz** [comment]
Looking for a fix for this! @gastongouron Did you find a solution?

**Comment by wingkwong** [comment]
fixed in 0.5.1
